app/javascript/components/patient/Patient.js

```diff
@@ -151,12 +151,11 @@ class Patient extends React.Component {
             />
           )}
           <Col sm={12}>
-            <h3>
-              <span aria-label={formatName(this.props.details)} className="pr-2">
-                {formatName(this.props.details)}
-              </span>
+            <h2 className="secondary-title">
```

Comment:
Header tags should always be sequential. The way that header tags looks doesn't matter, but the hierarchy is important.